src/fragments/lib-v1/storage/js/configureaccess.mdx

```diff
@@ -0,0 +1,94 @@
+Storage module can manage files with three different access levels; `guest`, `protected` and `private`. The Amplify CLI configures three different access levels on the storage bucket: guest, protected and private. When you run `amplify add storage`, the CLI will configure appropriate IAM policies on the bucket using a Cognito Identity Pool Role. You will have the option of adding CRUD (Create/Update, Read and Delete) based permissions as well, so that Authenticated and Guest users will be granted limited permissions within these levels.
+
```

Comment:
here it's called "guest" level access but in the "List files" page, it's called "public". What's the correct term? (I thought public)

Reply:
Made this consistent